Integrations/integration-aws.yml

```diff
@@ -4,7 +4,6 @@ commonfields:
 name: Amazon Web Services
 display: Amazon Web Services
 category: IT Services
-system: true
```

Comment:
the system removed it when I downloaded the script. It's not needed anymore...  (anything that comes from us is automatically tagged as system as far as I know)